pyproject.toml

```diff
@@ -365,7 +365,9 @@ minversion = "6.0"
 addopts = "-ra --doctest-glob='*.rst'"
 testpaths = ["docs/conversion_examples_gallery/", "tests"]
 doctest_optionflags = "ELLIPSIS"
-
+filterwarnings = [
+    "ignore:Date is missing timezone information. Updating to local timezone.:UserWarning:pynwb.file"
+]
```

Comment:
What is the purpose of this? It seems separate from the issue you are trying to solve, isn't it?

Reply:
Probably a merge mistake I made:
https://github.com/catalystneuro/neuroconv/pull/1240